packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerModule.java

```diff
@@ -488,40 +490,50 @@ public void run() {
                   .setOutputCompressQuality(quality == null ? DEFAULT_QUALITY : quality)
                   .start(getExperienceActivity());
             } else {
-              final Bitmap.CompressFormat finalCompressFormat = compressFormat;
-              mImageLoader.loadImageForManipulationFromURL(uri.toString(), new ImageLoader.ResultListener() {
-                @Override
-                public void onSuccess(@NonNull Bitmap bitmap) {
-                  int width = bitmap.getWidth();
-                  int height = bitmap.getHeight();
-
-                  ByteArrayOutputStream out = base64 ? new ByteArrayOutputStream() : null;
+              // No modification requested
+              if (quality == null || quality == DEFAULT_QUALITY) {
```

Comment:
Let's hardcode `100` here, in case someone changes `DEFAULT_QUALITY` to something like `80`.